### PR TITLE
Report code coverage to CodeClimate

### PR DESCRIPTION
**Pull request by jgsmith-usds**
**Why**:
We want to track code coverage for our tests and
make it part of the acceptance criteria for a PR.

**How**:
Configure the CodeClimate integration and upload
coverage reports.

### CODE REVIEW

**Comment by jmhooper** [comment]
Is that last commit for the graph construction stuff supposed to be in here?

**Comment by jgsmith-usds** [comment]
I was trying to fix a problem that CodeClimate found, but figured rake tasks aren't a big maintenance problem even if they are a bit wordy. I'm going to try and either fix it or remove rake tasks from the check.
